core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/async/CombinedResponseAsyncHttpResponseHandler.java

```diff
@@ -68,6 +68,7 @@ public void onError(Throwable error) {
         }
 
         successResponseHandler.onError(error);
+        errorResponseHandler.onError(error);
```

Comment:
On line 66, does this comment apply to the errorResponseHandler as well? Should we include a null check here for errorResponseHandler